internal/verification/database/health_authority_db_test.go

```diff
@@ -77,7 +77,7 @@ func TestAddRetrieveHealthAuthorityKeys(t *testing.T) {
 	}
 
 	wantKeys := []*model.HealthAuthorityKey{
-		&model.HealthAuthorityKey{
+		{
```

Comment:
covered in https://github.com/google/exposure-notifications-server/pull/502

Reply:
Done